### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mrsimonfletcher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # test
 
 my testing 
+
```
[the remark this answers is not in the document]
Comment (callmefletch) [reply README.md]: I do indeed.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # test
```
Comment (mrsimonfletcher) [inline README.md]: @callmefletch do you get this single comment?

Comment (mrsimonfletcher) [reply README.md]: @callmefletch what about this one?

Comment (mrsimonfletcher) [reply README.md]: @callmefletch what about this one?

**Pull request closed without merging**